openmc/filter.py

```diff
@@ -781,12 +781,18 @@ def get_pandas_dataframe(self, data_size, distribcell_paths=True):
             filter_bins = np.repeat(self.bins, self.stride)
             tile_factor = data_size / len(filter_bins)
             filter_bins = np.tile(filter_bins, tile_factor)
-            filter_bins = [x if x != 1 else 'x-min' for x in filter_bins]
-            filter_bins = [x if x != 2 else 'x-max' for x in filter_bins]
-            filter_bins = [x if x != 3 else 'y-min' for x in filter_bins]
-            filter_bins = [x if x != 4 else 'y-max' for x in filter_bins]
-            filter_bins = [x if x != 5 else 'z-min' for x in filter_bins]
-            filter_bins = [x if x != 6 else 'z-max' for x in filter_bins]
+            filter_bins = [x if x != 1  else 'x-min out' for x in filter_bins]
+            filter_bins = [x if x != 2  else 'x-max out' for x in filter_bins]
+            filter_bins = [x if x != 3  else 'y-min out' for x in filter_bins]
+            filter_bins = [x if x != 4  else 'y-max out' for x in filter_bins]
+            filter_bins = [x if x != 5  else 'z-min out' for x in filter_bins]
+            filter_bins = [x if x != 6  else 'z-max out' for x in filter_bins]
+            filter_bins = [x if x != 7  else 'x-min in'  for x in filter_bins]
+            filter_bins = [x if x != 8  else 'x-max in'  for x in filter_bins]
+            filter_bins = [x if x != 9  else 'y-min in'  for x in filter_bins]
+            filter_bins = [x if x != 10 else 'y-max in'  for x in filter_bins]
+            filter_bins = [x if x != 11 else 'z-min in'  for x in filter_bins]
+            filter_bins = [x if x != 12 else 'z-max in'  for x in filter_bins]
```

Comment:
I think this would be cleaner with a dictionary and a list comprehension:
```Python
current_name = {1: 'x-min out', 2: 'x-max out', ...}
filter_bins = [current_name[x] for x in filter_bins]
```
You could make the dictionary a private module-level variable.